deps.sh

```diff
@@ -4,7 +4,7 @@
 set -e
 
 # Test dependencies:
-tarantoolctl rocks install luatest 0.5.4
+tarantoolctl rocks install luatest 0.5.5
```

Comment:
If there is a group parametrization we probably should use it instead of pgroup.

Reply:
I know :) I will replace our own implementation with luatest's implementation as soon as I'll find a time for it.